integrations/tensorflow/bindings/python/pyiree/tf/support/tf_utils.py

```diff
@@ -209,6 +209,11 @@ def make_dims_dynamic(spec: tf.TensorSpec) -> tf.TensorSpec:
   return tf.TensorSpec([None] * len(spec.shape), spec.dtype)
 
 
+def keras_arg_wrapper(*args):
+  """Wrapper to convert multiple positional args into a list of values."""
+  return list(args) if isinstance(args, tuple) else args
```

Comment:
my python might be rusty, but I think that starargs will always be a tuple? Is this function actually dong anything vs just doing `list(args)`?

Reply:
Good catch! I assumed this was working because I assumed that `tf.keras.Model` had the same input behavior as `tf.keras.Layer`, and this would need to be if it did. (But it's not working and `tf.keras.Model` isn't as picky about what you give it).